roles/vmrole/templates/vm-template-windows2012r2.yaml.j2

```diff
@@ -58,6 +58,7 @@ objects:
           - name: disk0-pvc
             persistentVolumeClaim:
               claimName: windows-vm-pvc-${NAME}
+{% if not existing_pvc %}
```

Comment:
Can we achieve same semantics using OC templates?

Similar change shall go to https://github.com/kubevirt/kubevirt/blob/master/cluster/vm-template-windows2012r2.yaml and be pulled here until templates are available in the API.

Reply:
Since these are two tasks, how about use ansible and do an ```oc process -f ``` on each template, but have a ```when: not existing_pvc``` to select if the pvc template is run.

Reply:
no, oc templates dont have conditionals